modules/ROOT/Edits/endpoints.adoc

```diff
@@ -0,0 +1,21 @@
+= Endpoints
+
+Endpoints define the transport protocol configurations necessary to receive or send messages with external trading partners or internal backend applications.
```

Comment:
"to receive or send messages with" is not grammatical (s/b "receive from", "send to"). This is much simpler:
```suggestion
Endpoints define the transport protocol configurations necessary to communicate with external trading partners or internal backend applications.
```
```